### ISSUE TITLE
Ignored openvpn connection

### ISSUE THREAD

**Issue by Almtesh** [syncthing/syncthing-android#846]
Hello,

I configurated a opvn connection and the syncthing client does not use it.
Other applications use it, and syncthing on other devices too.

### Version Information

    App Version: 0.9.6
    Syncthing Version: v0.14.22-rc.1+12-g5a1ee7f
    Android Version: Android 5.1.1
    ovpn client: OpenVPN for Android 0.6.63

**Comment by licaon-kter** [comment]
Any limitation, on Wi-Fi or anything?

**Comment by Almtesh** [comment]
No, there's no network limitation. Actually, I put no limitation at all.

**Comment by ywum7fhv0skqh07h** [comment]
I'm having the opposite issue, where local traffic isn't working while OpenVPN is on. OpenVPN is set to ignore local traffic, yet Syncthing was getting no connection on Android. Setting OpenVPN to completely bypass Syncthing worked to fix my problem, but wouldn't be ideal if I was using Syncthing via the internet (I don't intend to).

**Comment by Almtesh** [comment]
Update, I don't know why, but syncthing uses the ovpn connection, but not the configurated DNS server in it.

**Comment by licaon-kter** [comment]
>  but not the configurated DNS server in it.

Seeing this too, basically Syncthing-Android tries to resolve DNS queries with the connection (Wi-Fi or mobile) DNS servers instead of the VPN DNS servers.

**Comment by Almtesh** [comment]
Now, I have another problem, syncthing ignores the openvpn IPv6 connection.
My openvpn connection does have IPv6 address and my peer name resolves IPv6 too.

I put a comment juste now because I added IPv6 just the last week.
